### ISSUE TITLE
window shows only desktop

### ISSUE THREAD

**Issue by SciEditor** [michelf/sim-daltonism#27]
No matter the app settings, it is showing only the desktop, not the open window it is on top of.
Sonoma 14 OS
Apple M2 Pro chip

**Comment by michelf** [comment]
This is macOS security preventing Sim Daltonism from seeing your screen. You can go to System Settings > Security & Privacy > Screen Recording, and toggle the permission for Sim Daltonism.

**Comment by ignaciotcrespo** [comment]
thanks for the tip, this should be in the FAQ inside the app :)

**Comment by psybers** [comment]
Maybe the app could be updated to detect that setting is not yet enabled, and prompt the user to go set it.

**Comment by psybers** [comment]
Ah, looks like #13 is requesting my suggestion.

**Comment by michelf** [comment]
It's certainly doable. In fact I accepted a [pull request](https://github.com/michelf/sim-daltonism/pull/20) that implements this a while ago (alongside a whole new filter system and some other things), but now I find there are changes I want to make in this code + I need to port it to iOS (for the new filter system) so it's all sort of stuck until I look at things more closely, make some decisions, and work on all these things. And I guess I feel kind of blocked by having too many things on my plate, and so nothing gets done.

Long story short: if you build the code from the master branch you'll see this prompt. It still needs work on the UI side IMO, plus I need to sort out all the other changes, and so there is no release yet.

**Comment by aarontheman** [comment]
In case anyone had some trouble finding the right spot:

![image](https://github.com/user-attachments/assets/996d3119-2eab-48b2-a46e-b123c256469e)
